integration-tests/suites/base.go

```diff
@@ -282,6 +282,24 @@ func (s *IntegrationTestSuiteBase) GetProcesses(containerID string) ([]common.Pr
 	return processes, nil
 }
 
+func (s *IntegrationTestSuiteBase) HasNetworks(containerID string) (bool, error) {
```

Comment:
I don't think this function is needed.

Reply:
GetNetworks() returns an error when there are no networks. Do you prefer to change this and return an empty set ?

Reply:
I implemented as suggested below, nevemind.